submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/repositories/SubmissionMongoRepository.kt

```diff
@@ -45,6 +45,8 @@ interface SubmissionDraftRepository : MongoRepository<DocSubmissionDraft, String
 
     fun deleteByUserIdAndKey(userId: String, key: String)
 
+    fun deleteByUserIdAndContentContaining(userId: String, content: String)
```

Comment:
is not the accession the `key`? I just wondering if there is a better way to perform this operation as full document scan looks pretty heavy